include/triton/Dialect/Triton/IR/TritonOps.td

```diff
@@ -1341,7 +1359,7 @@ def TT_DescriptorGatherOp : TT_Op<"descriptor_gather"> {
   }];
 }
 
-def TT_DescriptorScatterOp : TT_Op<"descriptor_scatter"> {
+def TT_DescriptorScatterOp : TT_Op<"descriptor_scatter", [TT_DescriptorOpInterface]> {
```

Comment:
Shouldn't this also be DescriptorStoreLikeOpInterface?

Reply:
done